src/main/java/nl/tudelft/ti2206/group9/gui/scene/SplashScene.java

```diff
@@ -105,7 +105,7 @@ private static void generateFadeTransition(final Label label) {
      * Every Button has an AudioPlayer for a sound effect.
      * @return the button AudioPlayer.
      */
-    public static AudioPlayer getButtonAudioPlayer() {
+    public static SoundEffectPlayer getButtonAudioPlayer() {
```

Comment:
I know this method exists for a while already, but I suggest to move this to `ShaftEscape` or `SoundEffectPlayer`. Scenes do not need to communicate in order to function correctly, which can be avoided here.

<review,interface,organization